src/coreclr/src/tools/crossgen2/crossgen2/crossgen2.csproj

```diff
@@ -4,7 +4,6 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <OutputType>Exe</OutputType>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
-    <TargetFramework Condition="'$(BuildingInsideVisualStudio)' == 'true'">netcoreapp3.0</TargetFramework>
```

Comment:
Do we know why this was needed at all?

Reply:
Yes, at the time when we switched to netcoreapp5.0 we added these lines as 5.0 wasn't supported in VS.

Reply:
Ok, cool. Thanks for clarifying.